src/main/java/picard/util/IntervalListTools.java

```diff
@@ -552,75 +551,72 @@ static class ScatterSummary {
     }
     /**
      * Method to scatter an interval list by locus.
+     * The scattered interval_list will have the following format (when SCATTER_COUNT = 3)
+     * - OUTPUT/temp_0001_of_3/scatter.interval_list
+     * - OUTPUT/temp_0002_of_3/scatter.interval_list
+     * - OUTPUT/temp_0003_of_3/scatter.interval_list (need to check)
      *
      * @param list The list of intervals to scatter
      * @return The scattered intervals, represented as a {@link List} of {@link IntervalList}
      */
-    private  ScatterSummary writeScatterIntervals(final IntervalList list) {
+    private ScatterSummary writeScatterIntervals(final IntervalList list) {
+        ValidationUtils.validateArg(SCATTER_COUNT > 0, "Scatter count should be a positive integer.");
```

Comment:
It's good to include the value in the error message i.e. "Scatter count should be a positive integer but it was " + SCATTER_COUNT + "." 

In methods which will be run millions of times you have to be careful about that sort of thing to avoid accidentally wasting cycles on useless too strings, but this is only going to happen ~once.